src/app/demos/affix/affix-demo.component.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  ViewChild
+} from '@angular/core';
+
+import {
+  SkyAffixHorizontalAlignment,
+  SkyAffixPlacement,
+  SkyAffixSubjectVisibilityChange,
+  SkyAffixVerticalAlignment
+} from '../../public';
+
+@Component({
+  selector: 'app-affix-demo',
+  templateUrl: './affix-demo.component.html',
+  styleUrls: ['./affix-demo.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AffixDemoComponent {
+
+  public placements: SkyAffixPlacement[] = ['above', 'right', 'below', 'left'];
+
+  public horizontalAlignments: SkyAffixHorizontalAlignment[] = ['right', 'center', 'left'];
+
+  public verticalAlignments: SkyAffixVerticalAlignment[] = ['bottom', 'middle', 'top'];
+
+  public model: {
+    placement: SkyAffixPlacement;
+    horizontalAlignment?: SkyAffixHorizontalAlignment;
+    verticalAlignment?: SkyAffixVerticalAlignment;
+  } = {
+    placement: 'above',
+    horizontalAlignment: 'center',
+    verticalAlignment: 'middle'
+  };
+
+  public isSticky: boolean = true;
+
+  public enableAutoFit: boolean = true;
+
+  public enableScrollableParent: boolean = false;
+
+  public disabled: boolean = false;
+
+  public isVisible: boolean = false;
```

Comment:
Check your alphabetical ordering of properties here.

Reply:
Good call. I'd like to keep the placement and alignment arrays near each other, but I'll go ahead and clean up the rest of the properties.